assets/controller.yaml

```diff
@@ -56,6 +55,10 @@ spec:
           env:
             - name: CSI_ENDPOINT
               value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
+            - name: CSI_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
             - name: AWS_ACCESS_KEY_ID
```

Comment:
Not to reviewers: This is the change that is required to make the pod work without metadata service.